mdx-models/src/main/java/com/mx/path/model/mdx/accessor/feature/FeatureMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.mx.path.model.mdx.accessor.feature;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.mx.path.core.common.request.Feature;
+
+public class FeatureMapper {
```

Comment:
I don't think we need a FeatureMapper class. The Feature enum contains these strings already. And it's overriding the toString() method to return the name. Could probably use that instead https://github.com/mxenabled/path-core/blob/master/common/src/main/java/com/mx/path/core/common/request/Feature.java#L113

Reply:
The feature should just be provided as a constructor arg. No need to map anything.